src/js/setUpSite.js

```diff
@@ -159,18 +161,44 @@ const generateScorecard = (scoreCardEntry) => {
 };
 
 /**
- * Move the map to the city boundaries and set its score card.
+ * Load city parking lots if not already loaded.
+ *
+ * @param cityId: E.g. `columbus-oh`.
+ * @param parkingLayer: GeoJSON layer with parking lot data
+ */
+const loadParkingLot = async (cityId, parkingLayer) => {
+  let load = true;
+  parkingLayer.getLayers().forEach((loadedCity) => {
+    if (loadedCity.feature.properties.id === cityId) {
+      load = false;
+    }
```

Comment:
I don't follow this. Instead, should the logic be 

```javascript
const city = parkingLayer.getLayers().find((city) => city.feature.properties.id == cityId);
load = city.feature.properties.<something>
```

Right now, this seems like it will always reload the same city no matter what, even if it was loaded before.

Reply:
`parkingLayer` is the layers of already loaded lots. 
So the logic right now is, if the city we want to load is already loaded (aka in `parkingLayer`), then don't load it.